starters/apps/basic/src/components/starter/gauge/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { component$ } from '@builder.io/qwik';
+import styles from './gauge.module.css';
+
+export default component$(({ value = 50 }: { value?: number }) => {
+  const safeValue = value < 0 || value > 100 ? 50 : value;
```

Comment:
let's inline this in the template, better performance for now

Reply:
i need the value as well on L25 (to calc the stroke array). still better to have the evaluation twice?